### DISCUSSION TITLE
Does it is useless code for codegen/main.go

### DISCUSSION THREAD

**Discussion by xiaods** [k3s-io/k3s#6192] [General]
https://github.com/k3s-io/k3s/blob/master/pkg/codegen/main.go#L14
```
var (
	basePackage = "github.com/k3s-io/k3s/types"
)
```

**Comment by brandond** [comment]
Looks like this has been unused since https://github.com/k3s-io/k3s/pull/466

**Reply by xiaods** [reply]
thanks a lot.
